Kconfig

```diff
@@ -1,2 +1,14 @@
-rsource "lvgl_tft/Kconfig"
-rsource "lvgl_touch/Kconfig"
+menu "LVGL ESP Drivers"
+
+    rsource "lvgl_tft/Kconfig"
+
+    rsource "lvgl_touch/Kconfig"
+
+    menu "I2C Port Settings"
+        depends on LV_I2C && !HAVE_I2C_MANAGER
+
+        rsource "i2c_manager/Kconfig"
+    
+    endmenu
```

Comment:
This menu disappears if I2C Manager the component is installed (`HAVE_I2C_MANAGER`).

In the odd case that it is not, but two components have it in 'OEM mode', they'll both show the menu showing the same settings (change in one immediately reflecting in other). But if you have to components that have I2C Manager you'd want to sync them up via the installed I2C Manager component, so this is not really an issue.

I guess it's fine to place it outside of the LVGL menu, I just figured it would be less confusing this way, but it may actually be better if they're always in the same place, even if there's an odd case where it's two menus.